airframe-http-recorder/src/main/scala/wvlet/airframe/http/recorder/HttpRecorderServer.scala

```diff
@@ -39,6 +39,22 @@ class HttpRecorderServer(recordStore: HttpRecordStore, finagleService: FinagleSe
     recordStore.record(request, response)
   }
 
+  def dumpSessionAsJson: String = {
+    recordStore.dumpSessionAsJson
+  }
+
+  def dumpAllAsJson: String = {
+    recordStore.dumpAllAsJson
+  }
+
+  def dumpSessionAsYaml: String = {
+    recordStore.dumpSessionAsYaml
+  }
+
+  def dumpAllAsYaml: String = {
+    recordStore.dumpAllAsYaml
+  }
```

Comment:
Added proxy methods to `HttpRecorderServer` for convenience because `HttpRecordStore` is not accessible depending on how `HttpRecorderServer` is created.

By the way, there is no difference between ReplayOnlyServer and ProgrammableServer any more? We may be able to consolidate them?
https://github.com/wvlet/airframe/blob/87f79c1284d6b0b20eecf9e3e0770a1c53be45b8/airframe-http-recorder/src/main/scala/wvlet/airframe/http/recorder/HttpRecorder.scala#L160-L166

https://github.com/wvlet/airframe/blob/87f79c1284d6b0b20eecf9e3e0770a1c53be45b8/airframe-http-recorder/src/main/scala/wvlet/airframe/http/recorder/HttpRecorder.scala#L172-L177

Reply:
Yeah. We can merge them into one

Reply:
I see. I will send another PR, then.